src/app/lib/access/dad_jokes/data/dad_joke_child_data.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:app/access/dad_jokes/data/dad_joke_content_data.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'dad_joke_child_data.g.dart';
+
+@JsonSerializable(createToJson: false)
+final class DadJokeChildData {
+  @JsonKey(name: 'data')
```

Comment:
Just curious, what does this do?

Reply:
It basically tells it to note generate code for toJson method because we don't need it here.

Reply:
the part I thought I highlighted was the @JsonKey(name: 'data') part

Reply:
Ohh it's for the deserialization part, the property is called 'data'.